src/main/java/com/team4/testingsystem/controllers/FileAnswerController.java

```diff
@@ -24,27 +22,25 @@ public FileAnswerController(FileAnswerService fileAnswerService) {
         this.fileAnswerService = fileAnswerService;
     }
 
-    @ApiOperation(value = "Is used to get a file answer from the database by its id")
-    @GetMapping(path = "/{id}")
-    public FileAnswer get(@PathVariable("id") long id) {
-        return fileAnswerService.getById(id);
+    @GetMapping("/essay/{testId}")
```

Comment:
the frontend shouldn't have a separate endpoint for getting essay. The data should be taken on our side and passed to DTO